proto/buf/alpha/registry/v1alpha1/repository_commit.proto

```diff
@@ -49,6 +49,12 @@ message RepositoryCommit {
   string spdx_license_id = 10;
   // The manifest digest of the commit.
   string manifest_digest = 11;
+  // Number of tags associated with the commit.
+  int64 tag_count = 12;
+  // Number of git commits with metadata associated with the BSR commit.
+  int64 git_commit_metadata_count = 13;
+  // Number of git commits without metadata associated with the BSR commit.
+  int64 git_commit_no_metadata_count = 14;
```

Comment:
I'm confused with this one, so if a commit has 2 git commits associated with it, it still has 2 git commit labels. Unless I'm missing something, I think `git_commit_no_metadata_count` is not necessary.

Reply:
I guess it's for the circumstance where the BSR commit has no git commit metadata associated with it, but still has git commit labels on it -- we would want to count those.

Reply:
Not sure how this is going to play out. For a BSR commit that was previously push, with, say 3 tags:

```
buf push --tag foo --tag bar --tag someGitCommitSHA1
```

and then sync runs, and the content is the same, attaching two git metadatas to the same BSR commit:

```
someGitCommitSHA2
someGitCommitSHA3
```

I guess the commit object would have:

```
{
  tag_count: 5, // foo, bar, and the 3 git shas
  git_commit_metadata_count: 2, // someGitCommitSHA2 and someGitCommitSHA3
  git_commit_no_metadata_count: 1 // someGitCommitSHA1
}
```

Am I right?